tests/test-a101-pins.js

```diff
@@ -26,18 +26,35 @@ for(var i = 0; i < GPIOPins.length; i++) {
     // IO6 and IO9 are defined but unusable as GPIOs currently
     if (pinName == "IO6" || pinName == "IO9") continue;
 
-    var pin = gpio.open({ pin: pins[pinName] });
-    var pinValue = pin.read();
-    assert(typeof pinValue == "boolean",
+    var pinA = gpio.open({ pin: pins[pinName], direction: "in" });
+    var pinAValue1 = pinA.read();
+    var ApullValue = (pinAValue1) ? " weak pull-up" : " weak pull-down";
+
+    assert(typeof pinAValue1 == "boolean",
+           "Arduino101Pins: " + pinName + ApullValue + " on input");
+
+    pinA.write(false);
+    var pinAValue2 = pinA.read();
+
+    assert(typeof pinAValue2 == "boolean" && pinAValue1 == pinAValue2,
            "Arduino101Pins: " + pinName + " input");
```

Comment:
OK, I think that makes sense. I have no idea why at this point. When I tried to set pullup/pulldown using Zephyr flags it never seemed to do anything. Maybe I need to try again. It could have something to do with pinmux settings.